aladdin/commands/build_components/__main__.py

```diff
@@ -74,7 +41,7 @@ def main():
         lamp=lamp,
         tag_hash=os.getenv("HASH", "local"),
         build_config=BuildConfig(),
-        components=sys.argv[1:],
+        components=sys.argv[2:],
```

Comment:
the command is not `aladdin build-components ...` so skip the first two args

Reply:
this is an interface breaking change, correct?

Reply:
hmm not quite, I think, outside the container the command is still `aladdin build-components` and inside the container `build-components` also still works

Reply:
as long as `aladdin build` still runs this code when `build_docker` is not specified in the `lamp.json` we should be ok

Reply:
yep still works 👍 I tried it on reporting-service